spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-auto-tracker.spec.js

```diff
@@ -128,13 +128,12 @@ describe('Google Analytics auto tracker', function () {
       expected.event = 'event_data'
       expected.event_data.event_name = 'select_content'
       expected.event_data.type = 'tabs'
-      expected.event_data.text = '/[date]/[postcode]/[email]'
+      expected.event_data.text = undefined
       expected.govuk_gem_version = 'aVersion'
 
       var attributes = {
         event_name: 'select_content',
-        type: 'tabs',
-        text: '/2022-02-02/SW10AA/email@example.com'
```

Comment:
Is this being removed because it's now handled somewhere else?

Reply:
@andysellick Whoops, I think I removed it for some reason when working on the regex and forgot to add it back. Will add this back in, thanks.